backspace-string-compare/src/lib.rs

```diff
@@ -0,0 +1,59 @@
+use std::collections::VecDeque;
+
+struct Solution {}
+impl Solution {
+    pub fn backspace_compare(s: String, t: String) -> bool {
+        let mut s_stack = VecDeque::new();
+        let mut t_stack = VecDeque::new();
+
+        s.chars().for_each(|letter| {
+            if letter == '#' {
+                s_stack.pop_front();
+            } else {
+                s_stack.push_front(letter);
+            }
+        });
+        t.chars().for_each(|letter| {
+            if letter == '#' {
+                t_stack.pop_front();
+            } else {
+                t_stack.push_front(letter);
+            }
+        });
```

Comment:
L9-L15 and L16-L22 are repetitive. Let's implement DRY (don't repeat yourself) and use functions.

Reply:
Thanks! done!